pkgs/misc/uboot/default.nix

```diff
@@ -1,15 +1,18 @@
 { stdenv
 , lib
-, fetchurl
-, fetchpatch
-, fetchFromGitHub
 , bc
 , bison
 , dtc
+, fetchFromGitHub
+, fetchpatch
```

Comment:
I think these can be removed? Are we using them?

Reply:
They're in use. They are used to fetch some vendor blobs, and patches.